source/PlanetPanel.cpp

```diff
@@ -319,6 +319,14 @@ void PlanetPanel::CheckWarningsAndTakeOff()
 	int missionCargoToSell = cargo.MissionCargoSize() - cargo.Size();
 	// Will you have to sell something other than regular cargo?
 	int cargoToSell = -(cargo.Free() + cargo.CommoditiesSize());
+	// Have you left any unique items at the outfitter?
+	bool hasUniques = false;
+	for(const auto &it : player.GetStock())
+		if(it.first->Category() == "Unique" || it.first->Attributes().Get("unique") > 0)
```

Comment:
I'd prefer if "uniqueness" were purely a function of the "unique" attribute, not including the category. I think that outfit categories should be a purely organizational piece, and we shouldn't have additional mechanics tied to specific categories.

We can leave it if others think it makes sense for the category alone to be a marker.